src/controllers/comment.controller.ts

```diff
@@ -69,6 +69,17 @@ class CommentController {
       next(error);
     }
   };
+
+  public likeComment = async (req: RequestWithUser, res: Response, next: NextFunction) => {
+    try {
+      const id = Number(req.params.comment_id);
+      const userId = req.user.id;
+      await this.commentService.likeComment({ id, userId });
+      res.status(200).json('Comment Liked');
```

Comment:
status code should be 201. On google it says 201 if the "request has succeeded and has led to the creation of a resource".